Configuration/PyReleaseValidation/python/MatrixReader.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, opt):
 
         self.wm=opt.wmcontrol
         self.revertDqmio=opt.revertDqmio
-        self.addCommand=opt.command
+        self.addCommand=' '.join(opt.command)
```

Comment:
@srimanob , by default opt.command is `None` and the above will fail if `--command` option is not used. I would suggest to revert this change and instead add the following
```
    if opt.command: opt.command = ' '.join(opt.command)
```
after https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/scripts/runTheMatrix.py#L307

Reply:
Thanks @smuzaffar 
I've updated the PR.